docs/guide/extract.md

```
@@ -717,6 +717,347 @@ Other limitations:
+
+    # Extract from public service
+    gpio.extract_arcgis('https://services.arcgis.com/.../FeatureServer/0') \
+        .sort_hilbert() \
+        .write('output.parquet')
+    ```
+
+### URL Format
+
+`SERVICE_URL` must be a full ArcGIS Feature Service layer URL including the layer ID:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)